distance_display/README.md

```diff
@@ -13,9 +13,18 @@ this repository.
 
 ![Widget In Action](distance_widget.png)
 
-There are three user-adjustable parameters in the widget.  The first, `how_many_to_show` sets the number of countries shown in the widget.  The second, `output_format` defines the distance output format.  This can be in miles,  feet, meters, kilometers, or rods.  The last parameter, `near_or_far` tells the widget to sort by nearest or most distant.
-
-The code uses the haversine distance equation to calculate distances.  There is also a javascript implmentation in the file `haversine.js`
```

Comment:
ooh, I am pleased by your changes to the second paragraph. Links & spelling correction FTW!